features/core.adoc

```diff
@@ -127,3 +127,91 @@ as the third and later words of the `WHAT` response.
 The second word of the `WHAT` response is a human-readable error
 message.  Future revisions of the spec *may* define a structure for
 this error message.
+
+== Resources
+
+=== `/`
+
+The _root resource_.  Every resource is a descendant of this resource.
+
+Behaves as a simple composite resource for all actions.
+
+=== `/info`
+
+The _information resource_, which contains _read-only_ information
+about the _service itself_.
+
+Behaves as a simple composite resource for `GET`.  All other actions
+are forbidden.
+
+=== `/info/serverid`
+
+Contains the server identifier that would be returned by the `OHAI`
+Internal API response.
+
+All actions apart from `GET` are forbidden.
+
+==== Example
+
+If a service identifies as `OHAI listd-0.3.0/playd-0.3.0`, then the
+`GET` response value for `/info/serverid` shall be
+`"listd-0.3.0/playd-0.3.0"`.
+
+=== `/info/features`
+
+Contains a list of strings representing the implemented features on
+the queried service (those returned by a `FEATURES` Internal API
+response from this service).
+
+All actions apart from `GET` are forbidden.
+
+=== `GET`
+
+`GET` responses *must* return a list of strings that correspond
+exactly to the names of the features implemented by the queried
+service.  There is _no_ defined ordering on the strings.
+
+==== Example
+
+If a service provides `FEATURES PlayStop FileLoad End`, the `GET`
+response value for `/info/features` *may* be `["PlayStop", "FileLoad",
+"End"]` (or any permutation thereof).
+
+=== `/ctl`
```

Comment:
as discussed, could expand it to "control" (no need to shorten it really)

still think that perhaps it should be merged with info